locks/lock_test.go

```diff
@@ -23,10 +24,16 @@ func TestWithLockNoop(t *testing.T) {
 }
 
 // A mock lock that returns an error on AcquireLock
-type ErrorOnAcquireLock struct {}
+type ErrorOnAcquireLock struct{}
+
 var ErrorOnAcquire = fmt.Errorf("error-on-acquire")
-func (lock ErrorOnAcquireLock) AcquireLock(terragruntOptions *options.TerragruntOptions) error { return ErrorOnAcquire }
-func (lock ErrorOnAcquireLock) ReleaseLock(terragruntOptions *options.TerragruntOptions) error { return nil }
+
+func (lock ErrorOnAcquireLock) AcquireLock(terragruntOptions *options.TerragruntOptions) error {
+	return ErrorOnAcquire
+}
+func (lock ErrorOnAcquireLock) ReleaseLock(terragruntOptions *options.TerragruntOptions) error {
+	return nil
+}
```

Comment:
How weird. A few lines above, `gofmt` is perfectly happy with one-liner functions and even aligns their curly braces, and yet here, it makes them multiline functions... Nothing to do about it, just idly commenting :)